int_experian/int_experian_sfra/cartridge/templates/default/edq/edqHeaders.isml

```diff
@@ -1,4 +1,7 @@
 <iscontent type="text/html" charset="UTF-8" compact="true"/>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
Is jQuery being used now where it wasn't earlier?

Reply:
It was located on the htmlHead.isml; but this ISML was removed for SFRA v4.4 compability and placed here.